tests/v2/validation/resource/terraform/v2charts/target/main.tf

```diff
@@ -0,0 +1,282 @@
+#Target cluster for v2 charts install
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+    }
+
+  }
+}
+
+provider "rancher2" {
+  api_url   = var.rancher_api_url
+  token_key = var.rancher_token_key
+}
+
+resource "rancher2_cluster_sync" "target-sync" {
+  cluster_id =  var.cluster_id
+  wait_catalogs = true
+  state_confirm = 30
```

Comment:
You're probably aware, but the wait interval for each iteration of the `state_confirm` is 5 seconds. So this will wait 150 seconds. Not sure if this is intended or if you only want to wait for 30 seconds.